src/widgets/widget-model.js

```diff
@@ -18,8 +19,30 @@ module.exports = cdb.core.Model.extend({
     'collapsed': false
   },
 
+  super: function (name) {
```

Comment:
hahaha It is! As I see the inheritance pattern, I've tried to implement the super methos (it is a super hack ;P) but node 0.10.XX can't run it :( so the tests are broken and I don't use it... so it is unnecessary. Let's remove it!